pkgdb2/api/packages.py

```diff
@@ -639,7 +660,9 @@ def api_package_unretire():
 @API.route('/package')
 @API.route('/package/<pkgname>/')
 @API.route('/package/<pkgname>')
-def api_package_info(pkgname=None):
+@API.route('/package/<namespace>/<pkgname>/')
+@API.route('/package/<namespace>/<pkgname>')
+def api_package_info(namespace='rpms', pkgname=None):
```

Comment:
But.. if it doesn't work when you visit ``/package/nethack``, how would that default value ever be provided?